CustomerApp/CustomerApp/Pages/YourOrderPage.xaml

```diff
@@ -0,0 +1,139 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             mc:Ignorable="d"
+             x:Class="CustomerApp.Pages.YourOrderPage"
+             BackgroundImageSource="YourOrderPage.png">
+    <ContentPage.Content>
+        <Grid
+            x:Name="yourOrderLayout">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="50" />
```

Comment:
I would be careful "hardcoding" values for widths and heights. What looks to be great on your screen may look off kilter on other screen sizes and resolutions. Consider looking into "*" and "auto" values for these!:)